src/variable.test.js

```diff
@@ -0,0 +1,46 @@
+import isWindowsMock from 'is-windows'
+import varValueConvert from './variable'
+
+beforeEach(() => {
+  isWindowsMock.__mock.reset()
+})
+
+test(`doesn't affect simple variable values`, () => {
+  isWindowsMock.__mock.returnValue = true
+  expect(varValueConvert('foo')).toBe('foo')
+})
+
+test(`converts a : into a ; on Windows`, () => {
+  isWindowsMock.__mock.returnValue = true
+  expect(varValueConvert('foo:bar')).toBe('foo;bar')
+})
+
+test(`converts a ; into a : on UNIX`, () => {
```

Comment:
No problem, done. That was my first approach but I followed the existing logic on `command.js`: https://github.com/kentcdodds/cross-env/blob/master/src/command.js#L15

You folks should also simplify that to be consistent :)